### DISCUSSION TITLE
Embeddable Commons graph visualizations

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#137] [General Suggestions]
### What is the problem that your suggestion solves?

I would like to mirror the Commons work types network graph and contributors Sankey graph on an external site.

### What solution might meet your needs?

I would like to embed live Commons visualizations in another site, potentially in an iframe. 

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [X] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by rebeccaringuette** [comment]
It would be great if that network graph can include works that cite the item, such as publications.

**Reply by codycooperross** [reply]
Hi Rebecca! The network graph does currently include works that cite the item in the aggregation of each work type. Is there a way the network graph isn't meeting your needs? Feel free to [create an additional discussion post](https://github.com/datacite/datacite-suggestions/discussions/new?category=general-suggestions) as well.

**Reply by rebeccaringuette** [reply]
I think the break is on the publisher side based on conversations with AGU.
